lib/checkPullRequestReview.js

```diff
@@ -84,6 +84,77 @@ const handleChangesRequested = async (context, pullRequest) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+const handleApprovalByAllReviewers = async (context, pullRequest) => {
+  // Since the pull request has been approved, we add the LGTM label.
+  const labels = pullRequest.labels.map((label) => label.name);
+  if (!labels.includes(LGTM_LABEL)) {
+    await context.github.issues.addLabels(
+      context.repo({
+        issue_number: pullRequest.number,
+        labels: [LGTM_LABEL],
+      })
+    );
+  }
+
+  // Check if author can merge PR. All members can merge the pull request.
+  const authorCanMerge = await utilityModule.isUserAMemberOfTheOrganisation(
+    context, pullRequest.user.login
+  );
+
+  if (authorCanMerge) {
+    // Assign author.
+    await context.github.issues.addAssignees(
+      context.repo({
+        issue_number: pullRequest.number,
+        assignees: [pullRequest.user.login],
+      })
+    );
+    await context.github.issues.createComment(
+      context.repo({
+        issue_number: pullRequest.number,
+        body:
+          'Hi @' +
+          pullRequest.user.login +
+          ', this PR is ready to be merged. Please make sure there are no ' +
+          'pending comments before merge. Thanks!',
+      })
+    );
+  } else {
+    // Assign project owner to merge the pull request.
+    // A pull request will always have a changelog label because we are
+    // assigning the PR author to the pull request if it is created without
+    //  changelog label.
+    const changelogLabel = utilityModule.getChangelogLabelFromPullRequest(
+      pullRequest
+    );
+    const projectOwner = utilityModule.getProjectOwnerFromLabel(
+      changelogLabel
+    );
+    await context.github.issues.addAssignees(
+      context.repo({
+        issue_number: pullRequest.number,
+        assignees: [projectOwner],
+      })
+    );
+    await context.github.issues.createComment(
```

Comment:
Add a comment to explain that why we are sure of always having a changelog label.

Reply:
The comment was added at the top.